src/components/TreesMap/3d/trees-3d-layer.ts

```diff
@@ -0,0 +1,185 @@
+import mapboxgl from 'mapbox-gl';
+
+export const trees3DHighlightSourceId = 'trees-3d-highlight-source';
+export const trees3DHighlightLayerId = 'trees-3d-highlight-layer';
+export const trees3DCylinderSourceId = 'trees-3d-cylinders-source';
+export const trees3DCylinderLayerId = 'trees-3d-cylinders-layer';
+export const trees3DLayerId = 'trees-3d-layer';
+export const treeModelId = 'tree-general-model';
+
+const MAPBOX_TREE_CYLINDERS_TILESET_URL =
+  process.env.NEXT_PUBLIC_MAPBOX_TREE_CYLINDERS_TILESET_URL;
+const MAPBOX_TREE_CYLINDERS_LAYERNAME =
+  process.env.NEXT_PUBLIC_MAPBOX_TREE_CYLINDERS_LAYERNAME;
+
+export const trees3DLayer = {
+  id: trees3DLayerId,
+  //@ts-ignore
+  type: 'model',
+  source: 'trees',
+  'source-layer': 'trees',
+  layout: {
+    //@ts-ignore
+    'model-id': treeModelId,
+  },
+  paint: {
+    //@ts-ignore
+    'model-scale': [0.008, 0.008, 0.008],
+    'model-translation': [0, 0, 0],
+    'model-opacity': 1,
+  },
+};
+
+export const trees3DCylinderSource = {
+  id: trees3DCylinderSourceId,
+  type: 'vector',
+  url: MAPBOX_TREE_CYLINDERS_TILESET_URL,
+  minzoom: 0,
+  maxzoom: 20,
+  promoteId: 'id',
+};
+
+export const trees3DCylinderLayer = {
+  id: trees3DCylinderLayerId,
+  type: 'fill-extrusion',
+  source: trees3DCylinderSourceId,
+  'source-layer': MAPBOX_TREE_CYLINDERS_LAYERNAME,
+  paint: {
+    'fill-extrusion-height': 8,
+    'fill-extrusion-opacity': 0,
+    'fill-extrusion-color': '#ff0000',
+  },
+};
+
+export const trees3DHighlightSource = function (
```

Comment:
If you look through the codebase(s) we normally do

```js
export const foo = () =>
```

or

```js
export function foo(){}
```

but not `const foo = function()…`